analyzers/src/SonarAnalyzer.Common/SonarAnalyzer.Common.csproj

```diff
@@ -8,6 +8,12 @@
     <ProjectGuid>{8a8a663e-1318-4361-bc97-2e63666feadb}</ProjectGuid>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
what is this about?

should it be removed?

Reply:
Autogenerated noise